brian2/devices/cpp_standalone/templates/main.cpp

```diff
@@ -20,6 +21,13 @@
 
 #include <iostream>
 #include <fstream>
+#include <string>
+#include <sstream>
+#include <vector>
+
+Estimator estimator;
```

Comment:
I think a better alternative to the global variable here would be to put it into the report function with static scope. I guess just adding
```C++
static Estimator estimator;
```
to the beginning of the function would work. This would have the advantage that there's no `estimator` object when we don't use the report function in the first place.

Reply:
Noted. Thanks